commands/info/info.go

```diff
@@ -76,3 +81,35 @@ func Action(c *cli.Context) error {
 	}
 	return nil
 }
+
+func readX509FromFileOrURL(path string) (*x509.Certificate, error) {
+	// Check if given URL is a valid URI
+	if u, err := url.ParseRequestURI(path); err == nil {
+		resp, err := http.Get(u.String())
```

Comment:
@hionay Do we really need to send a get request or just an tls.Dial is enough? Also we should get the last cert in chain, not the first one. Something like:

```go
cfg := tls.Config{}
conn, err := tls.Dial("tcp", "google.com:443", &cfg)
if err != nil { ... }
certChain := conn.ConnectionState().PeerCertificates
cert := certChain[len(certChain)-1]
```

Reply:
yes, `tls.Dial` is enough but if we don't use a URL flag we must use a scheme like `https://`. 
`url.ParseRequestURI` wants a scheme to parse. 

`url.Parse` doesn't require a scheme but in that case it conflicts with a file name.

if `url.Parse` is used, that will be a valid url (cert.pem):
```bash
gossl info cert.pem
```
and command will do a `tls.Dial` to `cert.pem` 

also port can be another port other than `443`

there are 2 options we can do:
```bash
gossl info https://google.com
```
or
```bash
gossl info --url google.com
```
@yakuter what's your opinion?


From what I've seen on several web sites, the first certificate is shown default on browsers not the last one:

google:
![image](https://user-images.githubusercontent.com/5264068/163731630-258588ae-d183-497e-a815-ebaa2a4f9be0.png)
![image](https://user-images.githubusercontent.com/5264068/163731633-fb156c2c-dc14-4d27-848b-1972105cb6ba.png)

my website:
![image](https://user-images.githubusercontent.com/5264068/163731618-8b3d4af1-961b-4f7b-81cd-6e3cc6cf3bf9.png)
![image](https://user-images.githubusercontent.com/5264068/163731622-3e3d1884-ffe3-4ecd-81b5-3fc011c2b751.png)

Reply:
I understand the issue now. Let's use --url flag for url but I don't want any flag for cert file. It should be still an argument. I want to keep it simple like `gossl info cert.pem`

The logic is, if url flag is set, then ignore the cert file argument and check url. I know it seems complicated a little bit but this is a trade-off I am willing to take.

One last thing, I need to do some research about choosing the right cert. Let's check the first certificate in chain for now as you already did.

Thank you.

Reply:
@yakuter I added `--url` flag according to this logic.

Reply:
if we don't want to use --url flag, we can add a check for file existance.

if an argument given, command will check if that file exists. If it does not exist, command will try to parse it as a URL and do a tls.Dial. 

@yakuter what do you think?

Reply:
Hmm this is also a nice way to check but we can also miss something while checking file. Let me also think about it. Maybe I can do some POC and see how it goes. Thank you for the idea @hionay